src/sql/workbench/contrib/query/browser/keyboardQueryActions.ts

```diff
@@ -227,9 +227,9 @@ export class EstimatedExecutionPlanKeyboardAction extends Action {
 	}
 }
 
-export class RunCurrentQueryWithActualPlanKeyboardAction extends Action {
-	public static ID = 'runCurrentQueryWithActualPlanKeyboardAction';
-	public static LABEL = nls.localize('runCurrentQueryWithActualPlanKeyboardAction', "Run Current Query with Actual Plan");
+export class ToggleActualPlanKeyboardAction extends Action {
+	public static ID = 'ToggleActualPlanKeyboardAction';
+	public static LABEL = nls.localize('ToggleActualPlanKeyboardAction', "Enable/Disable Actual Execution Plan");
```

Comment:
Is there any way that a user will know what state the editor is currently in? It would be even better if you could make these actions update to only show what is going to happen when it's clicked (and update the icon as well) if it isn't doing something like that already.

Reply:
At the moment, the only editor state indicator that is present to the user is the label on this button: 
![image](https://user-images.githubusercontent.com/87730006/176257932-29d519a7-f5b8-4723-9104-dd966625e013.png)

The button changes it's label from "Disable Actual Plan" to "Enable Actual Plan" and vice versa and only applies to the editor that was active at the time that the command was executed.